BurnOutSharp/ProtectionType/BD+.cs

```diff
@@ -33,24 +33,21 @@ public string CheckPath(string path, bool isDirectory, IEnumerable<string> files
             return null;
         }
 
+        // Version detection logic from libbdplus was used to implement this
         private static string GetVersion(string path)
         {
-            // Version detection logic from libbdplus was used to implement this
             try
             {
                 using (var fs = File.OpenRead(path))
                 {
                     fs.Seek(0x0d, SeekOrigin.Begin);
-                    int byte1 = fs.ReadByte();
-                    fs.Seek(0x0e, SeekOrigin.Begin);
-                    int byte2 = fs.ReadByte();
-                    fs.Seek(0x0f, SeekOrigin.Begin);
+                    int year1 = fs.ReadByte();
+                    int year2 = fs.ReadByte();
                     int month = fs.ReadByte();
-                    fs.Seek(0x10, SeekOrigin.Begin);
                     int day = fs.ReadByte();
 
-                    byte1 = byte1 << 8;
-                    int year = byte1 | byte2;
```

Comment:
You can combine these lines like the other source. If you're concerned about order of operations, just put the first statement in parentheses